packages/currency-validator/src/currency-validator.ts

```diff
@@ -0,0 +1,37 @@
+export interface FieldValidatorInterface {
+  keydown(e: KeyboardEvent): void;
+}
+
+export class CurrencyValidator implements FieldValidatorInterface {
+  keydown(e: KeyboardEvent): void {
+    const char = e.key;
+
+    // if user is holding down command/ctrl for like select all, allow it
+    if (e.metaKey) {
```

Comment:
`metaKey` isn't used in every OS for browser edit commands like select all, copy, paste, etc. We should also exit early for key combinations that use `char === 'Control'` as well.

Reply:
I will file a follow-up ticket for this. This is an existing issues and I'm just trying to extract this module from the main donation form.